service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/OrderQuestionBlock.java

```diff
@@ -12,14 +12,25 @@
         List.of("judgeAndLegalAdvisor")),
     APPROVAL_DATE("approvalDate", "Approval Date", OrderSection.ISSUING_DETAILS,
         List.of("manageOrdersApprovalDate")),
+    APPROVAL_DATE_TIME("approvalDateTime", "Approval Date Time", OrderSection.ISSUING_DETAILS,
+        List.of("manageOrdersApprovalDateTime")),
     WHICH_CHILDREN("whichChildren", "Which children", OrderSection.CHILDREN_DETAILS,
         List.of("orderAppliesToAllChildren", "children_label", "childSelector")),
+    EPO_TYPE("epoOrderType", "EPO Type", OrderSection.ORDER_DETAILS, List.of("manageOrdersEpoType")),
```

Comment:
As mentioned before I think we can treat all epo fields in the issuing details section as one question block